### PR TITLE
docs: fix typo in Hierarchical injectors doc

### PR DESCRIPTION
**Pull request by QiuranHu**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/hierarchical-dependency-injection.md [hunk 1]
```diff
@@ -741,7 +741,7 @@ The example application uses `@Optional()` so the application does not throw an
 The `<app-child>` currently provides the `AnimalService` in the `viewProviders` array with the value of dog <code>&#x1F436;</code>.
 Because the injector has only to look at the `ElementInjector` of the `<app-child>` for the `AnimalService`, it never sees the whale <code>&#x1F433;</code>.
 
-As in the `FlowerService` example, if you add `@SkipSelf()` to the constructor for the `AnimalService`, the injector won't look in the  `ElementInjector` or the current `<app-child>` for the `AnimalService`.
+As in the `FlowerService` example, if you add `@SkipSelf()` to the constructor for the `AnimalService`, the injector won't look in the  `ElementInjector` of the `<app-child>` for the `AnimalService`.
```
Comment (JeanMeche) [inline aio/content/guide/hierarchical-dependency-injection.md:744]: nit: It feels like the word current has his place here !

Comment (QiuranHu) [reply aio/content/guide/hierarchical-dependency-injection.md]: You're welcome. I've add the word current back.

**Review by JeanMeche — APPROVED** [review]

**Review by AndrewKushnir — APPROVED** [review]

**Comment by AndrewKushnir** [comment]
@QiuranHu thanks for improving the docs! Could you please also merge all commits into one, so that we can proceed with the merge?

**Comment by QiuranHu** [comment]
> @QiuranHu thanks for improving the docs! Could you please also merge all commits into one, so that we can proceed with the merge?

@AndrewKushnir Sure. I have merged all commits into one commit. Please review again.

**Review by AndrewKushnir — APPROVED** [review]

**Comment by AndrewKushnir** [comment]
@QiuranHu thanks for updating the PR 👍 I'm adding this PR to the merge queue.

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit ea4d9e8ef81a4d6aa5fee09b09ecadf12773ce77.

**Pull request closed without merging**